### PR TITLE
Add prerequisite and policy url configuration

### PR DESCRIPTION
**Pull request by thnilsen**
Please find suggestion for to new options.

1. Provide the policy URL during installation. 
2. Prerequisite checks during install. 

For the policy setting, if none is given during installation, the default policy.php is used. The github icon on the footer is also moved to the left to prevent it looking like part of the Imprint/Privacy policy.

image
image

The prerequisite checks during install that will check for the following:

* PHP Version (set to min 5.4 in this commit)
* Presence of mysqlnd library
* Write access to web directory

image

### CODE REVIEW

**Comment by Pryx** [comment]
Sorry for the delay, is this ready for review / merge?

**Comment by thnilsen** [comment]
I believe it should be ready for reivew yes, if you're happy with they way it looks and it put together

File: install.php [hunk 1]
```diff
@@ -138,6 +142,47 @@
...
+
+	$preq_phpver = $preq_fail;
+	$preq_mysqlnd = $preq_fail;
+	$preq_writedir = $preq_fail;
+
+	// Check if PHP version if > MINIMUM_PHP_VERSION
+	if (strnatcmp(phpversion(), MINIMUM_PHP_VERSION) >= 0) { $preq_phpver = $preq_ok; }
+
+	// Check if we have mysqlnd installed
+	if ( function_exists('mysqli_get_client_stats') ) { $preq_mysqlnd = $preq_ok; }
```
Comment (Pryx) [inline install.php]: Is this enough to check for mysqlnd? Any specific reason for checking this function? If so please add comment with explanation

Comment (thnilsen) [reply install.php]: I believe it is sufficient method to establish that mysqlnd is loaded. The mysqlnd driver provides some extra functions that is not available if the plain mysql package is installed, and `mysqli_get_client_stats` is one of them. This is documented on the PHP site at http://www.php.net/manual/en/mysqlnd.stats.php.

I also verified this on a Centos 7 install by removing the mysqlnd package and installing the plain mysql. With that package the test will fail. 

Another reference I found was this on stackoverflow - https://stackoverflow.com/questions/1475701/how-to-know-if-mysqlnd-is-the-active-driver

Comment (thnilsen) [reply install.php]: Comment added to install.php

**Pull request merged**